src/muya/lib/contentState/arrowCtrl.js

```diff
@@ -47,6 +47,11 @@ const arrowCtrl = ContentState => {
     }
     return null
   }
+  
+  ContentState.prototype.getTable = function (cell) {
```

Comment:
![1534593672472](https://user-images.githubusercontent.com/9712830/44299067-9a227d80-a321-11e8-825a-5ace3c741fbe.jpg)

if table is not the out most block, there will be problem. the cursor is at `1` position, and if you press arrow down, the cursor will jump to `2` position.

Reply:
I've updated getTable, now it takes first parent table